tests/pytorch/test_onnx_export.py

```diff
@@ -50,7 +50,9 @@
     from polygraphy.comparator import RunResults
 
 # The directory where generated ONNX test models are stored.
-TEST_ARTIFACTS_DIR = os.path.join(tempfile.gettempdir(), "./gen_onnx_models")
+TE_TEST_ARTIFACTS_DIR = os.environ.get('TE_TEST_ARTIFACTS_DIR')
```

Comment:
TE environment variables should be prefixed with `NVTE_`:
```suggestion
NVTE_TEST_ARTIFACTS_DIR = os.environ.get('NVTE_TEST_ARTIFACTS_DIR')
```

Reply:
All instances updated